ci/check_installed_data.py

```diff
@@ -12,7 +12,7 @@
 DATA_FILES_PATH = Path(DATA_FILES_PATH)
 assert DATA_FILES_PATH.is_dir(), DATA_FILES_PATH
 for subpath in (
-    "templates/page.html",
```

Comment:
As this is just one template out of many, I adjusted this to reduce risk of confusion with singleuser/template/page.html.